graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilder.java

```diff
@@ -145,19 +152,42 @@ public GraphqlServiceBuilder schema(GraphQLSchema schema) {
         return this;
     }
 
+    /**
+     * Sets {@link DataLoaderRegistry} creation function.
+     */
+    public GraphqlServiceBuilder dataLoaderRegistry(
+            Function<? super ServiceRequestContext, ? extends DataLoaderRegistry> dataLoaderRegistryFactory) {
+        checkArgument(dataLoaderRegistryConsumers == null,
+                "configureDataLoaderRegistry() and dataLoaderRegistry() are mutually exclusive.");
```

Comment:
Could you check the indentation of this file?

Reply:
Oops.. 🤣